test/e2e/pipeline/infra/cluster-api/setup.sh

```diff
@@ -37,11 +37,11 @@ source "$WORKDIR/cni.sh"
 
 CLUSTER_NAME=cluster
 
-export K8S_VERSION=${K8S_VERSION:-"1.25.5"}
+export K8S_VERSION=${K8S_VERSION:-"1.27.4"}
 K8S_VERSION=$(echo -n "$K8S_VERSION" | sed 's/v//g') # remove the leading v
 
 export CRI_PATH="/var/run/containerd/containerd.sock"
-export NODE_VM_IMAGE_TEMPLATE="quay.io/capk/ubuntu-2004-container-disk:v${K8S_VERSION}"
+export NODE_VM_IMAGE_TEMPLATE="harbor.crownlabs.polito.it/capk/ubuntu-2004-container-disk:v${K8S_VERSION}"
```

Comment:
Could be a good idea to pass the os image as environment variable and add it to the test matrix?